aspnetcore/blazor/fundamentals/static-files.md

```diff
@@ -193,6 +196,62 @@ For examples of how to address the policy violation with Subresource Integrity (
 
 :::moniker-end
 
+:::moniker range=">= aspnetcore-10.0"
+
+### Fingerprint client-side static assets
```

Comment:
nit: Should be H2? Others like "Import map Content Security Policy (CSP) violations" and "ImportMap component" are H2